cmd/newrelic-infra/newrelic-infra.go

```diff
@@ -200,7 +201,7 @@ func initializeAgentAndRun(c *config.Config, logFwCfg config.LogForward) error {
 
 	// Command channel initialization.
 	ccService := cmdchannel.NewService(caClient, c, ffManager)
-	initCmdResponse, err := ccService.InitialFetch()
+	initCmdResponse, err := ccService.InitialFetch(context.Background())
```

Comment:
Should `context.Background()` here be the global `context` that can be cancelled? I think to do that might mean a bigger refactor though.

Reply:
Yes, the whole context approach seems wrong to me and it requires a significant refactor.

It gets created within `NewAgent()` when it should be created at `main()` and injected into the agent constructor.

I'm not addressing this scope here, as `InitialFetch` life span is tiny enough so the tradeoff btw impact and cost isn't worthy at the moment.